### ISSUE TITLE
old PCR Test should vanish

### ISSUE THREAD

**Issue by viisauksena** [corona-warn-app/cwa-documentation#742]
old PCR Tests are still in Frontpage, even if weeks old. 
If someone not deletes them actively the old PCR Test should move somewhere else, since the app should go in business as usual mode. (maybe ask for it) 
Otherwise the main view is blocked by an very old pcr positive test.
Or give some information on how to get a "genesenen certificate" instead, 
or the info that the pcr test - if you are healthy now - is no longer needed and no good for anything.

**Comment by Ein-Tim** [comment]
@viisauksena 

So, just to make sure I understood you right, if one has registered a PCR test which comes back positive & then warns others, the screen which is then shown should automatically be not shown anymore after ca. 2-3 weeks?

**Comment by Jana-Z** [comment]
It would be even more convenient to be able to convert the positive PCR test into a Genesenenzertifikat with a click of a button.
If this is technically not possible it would be good to move the PCR test somewhere else, since you cannot access the „Risiko-Ermittlung“ feature with a postive PCR test. It should not be deleted since oftentimes the positive PCR test in the app is the only way some test centers tell you the result.

**Comment by Ein-Tim** [comment]
This issue should be moved to the cwa-wishlist.

**Comment by steffmueller** [comment]
I would like to +1 for the proposed solution. 

Since I didn’t find any active communication, that the risk calculation will be paused - I was assuming there is a bug in displaying risk cards and was about to open a bugticket. 

Unless users will be notified and reminded to delete the positive test result - they might think they need the test-card again. E.g. to get their proof in form of Genesungszertifikat. 
Some might never delete the test, to proof that they have had a COVID infection, for documentation reasons, because they are unsure what happens after deleting,(even simply because they want to show off).

**suggested solution:**
As a user I want to get an information that risk calculation is paused for x days due to reasons, because I want to understand  that there will not be any tracking of risk encounters in future. 

As a user I want risk calculation to continue after 14 days of quarantine so that I don’t have to worry about getting missed risk encounters. 

As a user I want to get the information when the pcr is not longer needed and an option to move the active PCR  test to continue to use the app as originated. 


thanks a ton.

**Comment by Ein-Tim** [comment]
@larswmh 

Can you move this to the wishlist repo?
